### PR TITLE
Game class file and runner file

### PR DESCRIPTION
**Pull request by dylanconnolly**

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  def player_place_ship(ship)
+    placement = gets.chomp.gsub(/\s+/, "").upcase.scan(/../)
+    if @player_board.place(ship, placement) == nil
+      puts "Invalid coordinates. Reminder: a #{ship.name} must be placed using #{ship.length} valid coordinates."
+      print ">"
+      player_place_ship(ship)
+    else
+      @player_board.place(ship, placement)
+    end
+  end
```
Comment (hale4029) [inline lib/game.rb:38]: I like the refactor for place ship, removed similar code from the place_cruiser and place_submarine methods

File: lib/game.rb [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+  def game_end
+    if @computer.submarine.health + @computer.cruiser.health == 0
+      puts "~~~~You won!~~~~"
+    elsif @player_sub.health + @player_cruiser.health == 0
+      puts ":( :( :( The computer wins. :( :( :("
+    end
+    puts "Returning to main menu..."
+    main_menu
```
Comment (hale4029) [inline lib/game.rb:127]: Do we need to return them to the main menu here?

File: lib/game.rb [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+      print ">"
+      player_place_ship(ship)
+    else
+      @player_board.place(ship, placement)
+    end
+  end
+
+  def place_cruiser
+    @player_cruiser = Ship.new("Cruiser", 3)
+    require "pry"; binding.pry
```
Comment (hale4029) [inline lib/game.rb]: do we want to remove this pry?

**Review by hale4029 — CHANGES_REQUESTED** [review]
looks good!

File: lib/game.rb (hunk 1 above)
Comment (dylanconnolly) [reply lib/game.rb:38]: Yea, we'll likely need to remove the specific place_cruiser and place_submarine methods if we want to make the game dynamic and allow the user to create any number of ships, but this works for now.

File: lib/game.rb (hunk 3 above)
Comment (dylanconnolly) [reply lib/game.rb]: good catch, I'll remove the pry, commit, then push to this branch

File: lib/game.rb (hunk 2 above)
Comment (dylanconnolly) [reply lib/game.rb:127]: One of the specs for the end of the game is to return the user to the main menu. Once at the main menu they can quit the game to stop the game from running again.

File: test/game_test.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+require './lib/cell'
+require './lib/ship'
+require './lib/game'
+
+class GameTest < Minitest::Test
+
+  def setup
+    @board = Board.new
+    @game = Game.new
+    require "pry"; binding.pry
```
Comment (dylanconnolly) [inline test/game_test.rb]: I'm going to remove this pry also, was from early on

**Pull request merged**